controller/search_blackbox_test.go

```diff
@@ -409,75 +369,40 @@ func (s *searchControllerTestSuite) TestSearchFilter() {
 func (s *searchControllerTestSuite) TestSearchByWorkItemTypeGroup() {
 	s.T().Run(http.StatusText(http.StatusOK), func(t *testing.T) {
 		// given
```

Comment:
Nitpicking Line number 371 "// given" should be removed

Reply:
@DhritiShikhar fixed in 6be3938